get-cloudify.py

```diff
@@ -372,35 +389,35 @@ def execute(self):
                            version=self.version,
                            pre=self.pre,
                            virtualenv_path=self.virtualenv,
-                           requirement_files=self.withrequirements,
+                           requirement_files=self.with_requirements,
                            upgrade=self.upgrade)
         elif os.path.isdir(self.wheels_path):
-            lgr.info('Wheels directory found: "{0}". '
-                     'Attemping offline installation...'.format(
-                         self.wheels_path))
+            logger.info('Wheels directory found: "{0}". '
+                        'Attemping offline installation...'.format(
+                            self.wheels_path))
             try:
                 install_module(module=module,
                                pre=True,
                                virtualenv_path=self.virtualenv,
                                wheelspath=self.wheels_path,
-                               requirement_files=self.withrequirements,
+                               requirement_files=self.with_requirements,
                                upgrade=self.upgrade)
             except Exception as ex:
```

Comment:
so.. I think we shouldn't fall back to online if offline failed. I'd rather remove the "offline" thing completely.